### PR TITLE
fix Linux build instructions

### LINKED ISSUE #48: Readme step provides error
For Linux builds, the README.md gives the following steps:

    cd libgeotiff
    mkdir build && cd build
    ./autogen.sh

However, `autogen.sh` does not exist in the newly-created `build/` directory (and running `../autogen.sh` complains about missing files). Is `mkdir build && cd build` unnecessary (or did I just set something up incorrectly)?

### PR DESCRIPTION
**Pull request by neteler**
fixes #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To ask questions and to follow release announcements, subscribe at the [mailing
 ### Linux
 ```
 cd libgeotiff
-mkdir build && cd build
+mkdir build
```
Comment (rouault) [inline README.md:28]: `mkdir build` is unneeded then

Comment (neteler) [inline README.md:28]:
```suggestion
```

Comment (neteler) [reply README.md]: fixed

**Comment by rouault** [comment]
thanks

**Pull request merged**